SwissKnife/info/ExecutionEnvironment.py

```diff
@@ -1,18 +1,22 @@
 from enum import Enum
 
 
-class ExecutionEnvironment(Enum):
+class ExecutionEnvironment(str, Enum):
```

Comment:
black magic!

Reply:
Yes! thanks to `Black Magic` we inherit both `str` and `Enum` and as `str` is JSON serializable, the work is already done :1st_place_medal: